crates/network/src/ethereum/mod.rs

```diff
@@ -62,4 +63,8 @@ impl TransactionResponse for alloy_rpc_types_eth::Transaction {
     fn gas(&self) -> u128 {
         self.gas
     }
+
+    fn input(&self) -> Bytes {
```

Comment:
wdyt @DaniPopes Bytes or &[u8] ?

Reply:
&Bytes would be the best choice here

Reply:
I guess we can reasonably assume that every tx always has an input, so that makes sense